crates/astria-composer/src/executor/mod.rs

```diff
@@ -192,15 +193,19 @@ impl Executor {
     }
 
     /// Create a future to submit a bundle to the sequencer.
-    #[instrument(skip(self), fields(nonce.initial = %nonce))]
-    fn submit_bundle(&self, nonce: u32, bundle: Vec<Action>) -> Fuse<Instrumented<SubmitFut>> {
+    #[instrument(skip_all, fields(
+        nonce.initial = %nonce,
+        bundle_bytes = %bundle.curr_size,
+        rollup_actions = %telemetry::display::json(&bundle.rollup_counts)
```

Comment:
hm, let's leave this for now but this might end up being very noisy. I haven't thought about the all the events emitted by this future, but just a thought.